doc/imv-folder.1.txt

```diff
@@ -0,0 +1,36 @@
+/////
+vim:set ts=4 sw=4 tw=82 noet:
+/////
+:quotes.~:
+
+imv (1)
+=======
+
+Name
+----
+imv-folder - Open 'imv' for all images in a folder
```

Comment:
I think that if we use specialised terminology, it should be explained. Otherwise it's impossible to figure out what this command does without reading the code or looking at related issues.

```suggestion
imv-folder - Open 'imv' for all images in a folder

A "folder" is simply a directory which contains one or more images.
```

This is not ideal IMHO though, I see no reason to use specialised terminology, we could just say "directory" instead of using a dedicated term.

"folder" is also confusing, since many applications use this same term to describe mailboxes (include popular webmail services). In the MS world, pretty much any collection of any sort of item is called a "folder" too.

Reply:
I though folder and directory were synonyms for the most part. If @eXeC64 thinks this a showstopper I can rename everything to mention "directory" instead.

Not that the "folder" name is already used on the README: https://github.com/eXeC64/imv#slideshow